README.rst

```diff
@@ -0,0 +1,70 @@
+Image Modal XBlock
+==================
```

Comment:
As much as I like markdown, restructured text is a first-class citizen in Python/PyPI land

Trying to use markdown instead means we end up swimming against the current.